README.md

```diff
@@ -4,39 +4,32 @@
 [![CircleCI](https://circleci.com/gh/Azure/aks-engine/tree/master.svg?style=svg)](https://circleci.com/gh/Azure/aks-engine/tree/master)
 [![GoDoc](https://godoc.org/github.com/Azure/aks-engine?status.svg)](https://godoc.org/github.com/Azure/aks-engine)
 
-## Overview
-
-AKS-Engine leverages ARM (Azure Resource Manager) to build Kubernetes IaaS in Azure. AKS-Engine provides convenient tooling to quickly bootstrap clusters, and implements cluster provisioning and lifecycle operations for [AKS](https://docs.microsoft.com/en-us/azure/aks), Azure's managed Kubernetes service offering.
-
-More info, including a thorough walkthrough is [here](docs/aksengine.md).
+AKS-Engine is the easiest way to provision your own Kubernetes cluster on Azure.
```

Comment:
unmanaged Kubernetes Cluster perhaps?

Reply:
Thanks for the feedback! I definitely wanted to simplify the verbiage a bit here. "unmanagable" has an odd english connotation, meaning it is uncontrollable or chaotic. That's why I decided to go with "your own" instead: it gives a sense of personal ownership, but I agree it doesn't clarify well that this is a clear difference from AKS.

How about "AKS-Engine is the easiest way to provision a DIY Kubernetes cluster on Azure"? I believe that "DIY" promotes a form of creativity (and gives a sense of "unmanaged") while avoiding the chaotic/uncontrollable emotions from "unmanagable", AND it serves the same purpose (making a clear separation between AKS and AKS-Engine).

What do you think?

Reply:
+100 for DIY !